Fixed #100 Feature addition, Multiple instruments can searched

proritu opened the pull request:
Fixed #99 place_basket_order() raises exception for data type of price as float

proritu committed:
Update alice_blue.py

Fixed #99 place_basket_order() raises exception for data type of price as float

proritu committed:
Enhanced #100 Multiple instruments can not be searched

Enhanced #100 Multiple instruments can not be searched

proritu committed:
Update README.md

Updated with the example for searching multiple instrument for more than one underlying

krishnavelu commented on alice_blue/alice_blue.py:
```diff
@@ -570,7 +570,7 @@ def place_basket_order(self, orders):
                 if "price" not in i:
                     raise TypeError("Each element in orders should have key 'price' if its a limit order ")
-                if isinstance(i['price'], float):
+                if not isinstance(i['price'], float):
```
This change is already done in pull request #102. If I merge that then this will be in problems.
Please segregate the changes for both issues/PRs
Like PR #102 is for issue #99 and
PR #103 is for #100 
You are doing a good job.

proritu committed:
Enhanced #100 Multiple instruments can not be searched

Duplicate pull request found for the same changes, so this changes has been reverted

krishnavelu commented:
@proritu thank you for your contribution.
This PR is now merged to develop.
